db/re/item_db.txt

```diff
@@ -679,7 +679,7 @@
 1314,F_Tomahawk_C,Tomahawk,5,2,,0,200,,1,0,0x000444A2,63,2,34,4,0,0,7,{ bonus bAtkEle,Ele_Wind; },{},{}
 1315,F_Right_Epsilon_C,Light Epsilon,5,1,,0,229,,1,0,0x000444A2,63,2,34,4,1,0,7,{ bonus bAtkEle,Ele_Holy; bonus bStr,10; },{},{}
 1316,Adventure_Axe,Adventure Axe,5,0,,0,100,,1,0,0x00040420,63,2,2,1,1,0,6,{},{},{}
-1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,1,1,1,6,{},{},{}
+1317,Academy_Axe,Academy Axe,5,0,,1600,130,,1,1,0x00040420,63,2,2,30,1,1,6,{ bonus bMaxHP,200; bonus bAspdRate,5; bonus bUnbreakableWeapon,1; bonus bAspdRate,(-1*(BaseLevel/10)); bonus bMaxHP,(-40*(BaseLevel/10)); },{},{}
```

Comment:
```
// ID,AegisName,Name,Type,Buy,Sell,Weight,ATK[:MATK],DEF,Range,Slots,Job,Class,Gender,Loc,wLV,eLV[:maxLevel],Refineable,View,{ Script },{ OnEquip_Script },{ OnUnequip_Script }
```
Your weapon level (wLV) is 30.
The bonus can be write `bonus bAspdRate,5(-1*(BaseLevel/10));` etc..

Reply:
wLV = wear Level
eLV = Equip Level

no? O.O

Reply:
```
wLV: Weapon level.

---------------------------------------

eLV: Base level required to be able to equip.

maxLevel: Only able to equip if base level is lower than this.
```

Reply:
Ok, so I'll change it too!

Reply:
Atemo's example is correct, he just combined the two bonuses into one since it already gives a flat 5% bonus.

Reply:
Done. ✌️